apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -84,6 +84,10 @@ type APIManagerSpec struct {
 	PodDisruptionBudget *PodDisruptionBudgetSpec `json:"podDisruptionBudget,omitempty"`
 	// +optional
 	Monitoring *MonitoringSpec `json:"monitoring,omitempty"`
+	// +optional
+	SystemDatabaseTLSEnabled *bool `json:"systemDatabaseTLSEnabled,omitempty"`
```

Comment:
if it's not a problem, please move this to system spec and zync spec respectively. Not a big deal if you prefer this is a global.

Reply:
Don't believe it would be a problem to move. It's preference really. And it doesn't matter to me where it lives .